radix-engine/src/updates/cuttlefish.rs

```diff
@@ -1,12 +1,14 @@
 use radix_transactions::validation::*;
 
 use super::*;
-use crate::system::system_callback::*;
+use crate::{kernel::kernel::KernelBoot, system::system_callback::*};
 
 #[derive(Clone)]
 pub struct CuttlefishSettings {
     /// Add configuration for system logic versioning
     pub system_logic_update: UpdateSetting<NoSettings>,
+    /// Updating the always visible global nodes to include the account locker package.
+    pub always_visible_global_nodes_update: UpdateSetting<NoSettings>,
```

Comment:
I think probably calling this `kernel_version_update` might be a nicer naming scheme, in case we want to add other things to Kernel changes this update.

Reply:
Sounds good, going with that name.